.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
```

Comment:
What does "release created" mean? Pushing an annotated tag? Or is this not a git thing? (GitHub-only?)

Reply:
Yes, my understanding is that GH converts all tags into releases when pushed, and that annotated tags have release notes, non-annotated tags do not.

Or, you can make a release through the GH webapp or API. I'm not sure whether that adds any more metadata than just pushing a tag would. This API is documented [here](https://developer.github.com/v3/repos/releases/).